Client/Frontend/Autofill/CreditCard/ViewComponents/CreditCardInputField.swift

```diff
@@ -192,7 +192,7 @@ struct CreditCardInputField: View {
         .onChange(of: text) { [oldValue = text] newValue in
             handleTextInputWith(oldValue, and: newValue)
             viewModel.updateRightButtonState()
-        }
+        }.accessibilityLabel("\(fieldHeadline) field")
```

Comment:
Yes, it makes sense to read the label as written. As for whether or not it needs to say "field" at the end, I would defer to @anna-yeddi whether that word (or a different word) is needed, or perhaps the screen reader is already appending a name for the field.

Reply:
> Yes, it makes sense to read the label as written. As for whether or not it needs to say "field" at the end, I would defer to @anna-yeddi whether that word (or a different word) is needed, or perhaps the screen reader is already appending a name for the field.

Thank you for tagging me, @ewachowiak! 

Since the field has appropriate accessible role - being an input field (I assume), then we do not need to include the role in the label, since the role would be announced by a screen reader in the language of the user's OS. So feel free to save a word and skip the `field` altogether

Reply:
@imsanchit correct: keeping just the headline for the label

Reply:
Sure, will make the necessary changes.

Reply:
hey, updated the PR, if possible please have a look.